.github/workflows/release_windows.yml

```diff
@@ -88,24 +88,17 @@ jobs:
           $latestYml.files[0].size = $size
           $latestYml | ConvertTo-Yaml | Set-Content -Path signed/latest.yml
 
-      - name: Create draft release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
-        with:
-          tag_name: ${{ github.ref }}
-          release_name: Release ${{ github.ref }}
-          draft: true
-          prerelease: false
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.3.2
```

Comment:
it seems like this GHA is meant to be used with the 

```
on:
  release:
    types: [created]
```

trigger. will it work if we're using the tag push trigger? Also will it work if this is the first workflow to complete (i.e. that there is no draft release created yet) so it has to create it?

https://github.com/bruceadams/get-release

Reply:
I think the previous actions/create-release would work if we removed the `Release ` prefix on release_name right? did you push a tag to this branch to try and test that?

Reply:
If this is the first workflow to complete it won't work but it is basically impossible for it to be the first release. If it happens it's because the Linux one failed.
It takes around 20 minutes to complete while the Linux one is the first to create the release since it has no signing or anything, in 5 minutes we already have the draft release ready.

Reply:
> I think the previous actions/create-release would work if we removed the `Release ` prefix on release_name right? did you push a tag to this branch to try and test that?

no, didn't know about that. We can use it but I don't think will make any difference in the end.
Looking at the code I don't see how it is triggered only on `created`, on their github they have only an example.

Reply:
Tested and it creates a new release anyway. Looking for alternatives now.

Reply:
Tried using some logic to check for the release and it was failing anyway. 
I think we should just use the Get Release action here anyway, if you check the times, at the point that the action starts building the windows exe, all the linux files were built and uploaded to the release, macOS as well. Windows is always the last one on this so should be safe enough.
There is also the problem of our quota on Signpath that is reaching the limit during these tests.

Reply:
okay that works